docs/enterprise-guide/enterprise-cloud-guide/backups/README.md

```diff
@@ -8,36 +8,11 @@ keywords: backups
 
 # Backups
 
-Your Enterprise cloud data is backed up continuously and retained for 30 days.
-Within those 30 days we can restore your data to any point in time with a precision of 5 minutes, in case you need us to.
+*For a single source of truth, we moved this information* 
```

Comment:
If we put the section here as proposed below we can just drop this line.

Reply:
i think that it is important to link to the Backup Via GUI from the Enterprise Cloud Backup page as before there was the whole process documented and now it is missing there...

Reply:
@machisuji please check

Reply:
You're right, it's important to link there. I still think the sentence "For a single source of truth, we moved this information" can go as it doesn't add anything useful for users. By linking the section in the line below it is obvious that it has been moved for people who were used to the text there and for new users it doesn't matter that it was moved. They just see that it is linked and that's fine.

So I say keep line 21 (Please navigate to...) and drop line 19 (For a single source...)